mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -518,6 +503,36 @@ def deactivate(self):
         self.bus.emit(msg.forward(f"intent.service.skills.deactivate",
                                   data={"skill_id": self.skill_id}))
 
+    def handle_converse_ack(self, message):
```

Comment:
Should this be an internal method since overriding could suppress the `ack` response?

Reply:
thats the point, advanced skill may want to override and do some checks here to decide if they want to converse or not

Reply:
specifically, `can_handle` is of interest for advanced skills

Reply:
i suppose an advanced skill can just override `converse_is_implemented`, will make private

Reply:
That makes sense. So then it's just up to the core module to decide when to timeout the `ack` for misbehaving skills